src/TodoList.js

```diff
@@ -19,6 +19,6 @@ function TodoList() {
     )
```

Comment:
Again, nice work getting a grasp on the `.map` function and array with objects. Just watch your code styling and indentation, particularly in the `todoList` object. VSCode has the ability to "Format Document" for JS files, which will fix this stuff for you automatically! 

https://code.visualstudio.com/docs/editor/codebasics#_formatting